invenio_oaiserver/config.py

```diff
@@ -131,3 +131,54 @@
 
 OAISERVER_CONTROL_NUMBER_FETCHER = 'recid'
 """PIDStore fetcher for the OAI ID control number."""
+
+OAISERVER_DESCRIPTIONS = []
+"""Specify the optional description containers that can be used to express
+properties of the repository that are not covered by the standard response
+to the Identify verb.
+For further information see:
+http://www.openarchives.org/OAI/2.0/guidelines.htm
+
+The `eprints`, `oai_identifier` and `friends` description can be added using
+the helper functions in utils.py as follows:
+
+.. code-block:: python
+
+    from invenio_oaiserver.utils import eprints_description
+    from invenio_oaiserver.utils import friends_description
+    from invenio_oaiserver.utils import oai_identifier_description
+
+    OAISERVER_DESCRIPTIONS = [
+        eprints_description(metadataPolicy, dataPolicy,
+                            content, submissionPolicy),
```

Comment:
order...

Reply:
![](https://giphy.com/gifs/what-confused-cross-eyed-q5ZGrl0J65ivu)

Reply:
![](http://reactiongifs.me/wp-content/uploads/2013/09/brian-regan-follow-ikea-instructions.gif)

Reply:
3rd time's the charm!